netjsonconfig/backends/openwrt/schema.py

```diff
@@ -1024,59 +1024,57 @@
-                                "minLength": 16,
+                                "type": "object",
+                                "title": "Network Member",
+                                "allOf": [{"required": ["id", "ifname"]}],
+                                "properties": {
+                                    "id": {
+                                        "type": "string",
+                                        "title": "Network ID",
+                                        "maxLength": 16,
+                                        "minLength": 16,
+                                        "description": "Network ID to join",
+                                    },
+                                    "ifname": {
+                                        "type": "string",
+                                        "title": "Interface name",
+                                        "minLength": 1,
+                                        "maxLength": 10,
+                                        "description": "Name of zerotier interface",
+                                    },
+                                },
                             },
                         },
-                        "config_path": {
+                        "secret": {
                             "type": "string",
                             "propertyOrder": 4,
+                            "default": "{{secret}}",
                             "description": (
-                                "Path to the persistent configuration "
-                                "folder (for zerotier controller mode)"
+                                "Secret key of the zerotier client (network member), "
+                                "You can leave it as the default and OpenWISP will automatically determine it"
                             ),
                         },
-                        "copy_config_path": {
+                        # Hidden property
+                        "config_path": {
                             "type": "string",
                             "propertyOrder": 5,
-                            "enum": ["0", "1"],
+                            "options": {"hidden": True},
+                            "default": "/etc/openwisp/zerotier",
                             "description": (
-                                "Specifies whether to copy the configuration "
-                                "file to RAM ('0' - No, '1' - Yes), this prevents "
-                                "writing to flash in zerotier controller mode"
-                            ),
-                        },
-                        "port": {
-                            "type": "integer",
-                            "minimum": 1,
-                            "maximum": 65535,
-                            "default": 9993,
-                            "propertyOrder": 6,
-                            "description": "Port number of the zerotier service",
```

Comment:
why is this being removed?

Reply:
I was testing [these changes](https://github.com/openwisp/netjsonconfig/pull/285#discussion_r1299955746) and the [JSON schema hidden property](https://github.com/openwisp/netjsonconfig/blob/issue-284/add-support-for-zt-ifname/netjsonconfig/backends/openwrt/schema.py#L1069-L1090). Accidentally, I pushed my local changes. Thank you for pointing it out. I have now reverted these changes. Thanks!